linux-anvil-aarch64-cuda/Dockerfile

```diff
@@ -35,21 +35,24 @@ ADD https://loripsum.net/api /opt/docker/etc/gibberish
 # Naming convention changed with cos8 - see:
 # * https://lists.centos.org/pipermail/centos-devel/2019-September/017847.html
 # * https://www.centos.org/keys/#project-keys
-RUN if [ "$CENTOS_VER" -le "7" ]; then \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER} && \
-        rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-${CENTOS_VER}-aarch64; \
-    else \
+RUN if [ "$CENTOS_VER" = "centos7" ]; then \
```

Comment:
Is this equality check correct? I think we should use `==`  or `-eq` but I always need to check these on the command line, so please double check :D 

(Same comment for the remaining changes that involve an `if` clause).

Reply:
I googled this as well — you can use single equals for string comparison with the single [. The commands would also fail if these comparisons weren’t working.

Source: https://linuxize.com/post/how-to-compare-strings-in-bash/
> Use the = operator with the test [ command. Use the == operator with the [[ command for pattern matching.

Reply:
Yeah both work. Shells are strange. Generally agree `-eq` is clearer than `=` because the latter typically makes one think of single assignment

Reply:
Ah ok. Fair enough